mainhub/mainhub.ino

```diff
@@ -93,19 +117,63 @@ void update_game()
     cursor_index = 1; // Move to the second LED
     direction = 1; // Change the direction to right
   }
-  // Increment the cursor, but reset to zero if it is greater than the number of leds (Circle Path)
-  /*cursor_index = cursor_index + 1;
-  if (cursor_index >= NUM_LEDS)
-  {
-    cursor_index = 0;
-  }
-*/
+  //LEDs
+  //Light the cursor
+  leds[cursor_index] = rainbowColors[currentLevel];
+  FastLED.show();
+  // Now turn the LED off, then pause
+  leds[cursor_index] = CRGB::Black;
+  FastLED.show();
+
+   //Light the target range
+  leds[min_index] = CRGB::White;
+  FastLED.show();
+  leds[max_index] = CRGB::White;
+  FastLED.show();
+  
   Serial.print("Cursor Value: ");
   Serial.println(cursor_index);
 
 }
 
+void game_logic()
+//WIN CASE
+{   if (is_button_pressed) {
```

Comment:
Careful here. `Is_button_pressed` is true for as long as the button is down. 

If you only want to run once, you can check the previous value against the current value. 

You are immediately setting cursor_index to 0 which helps avoid the bug for now, but you may need more complicated logic in the future.